inventory/sample/group_vars/all/docker.yml

```diff
@@ -1,42 +1,46 @@
 ## Uncomment this if you want to force overlay/overlay2 as docker storage driver
 ## Please note that overlay2 is only supported on newer kernels
-
 #docker_storage_options: -s overlay2
 
 ## Uncomment this if you have more than 3 nameservers, then we'll only use the first 3.
-
-#docker_dns_servers_strict: false
+docker_dns_servers_strict: false
```

Comment:
All variables in groups_vars  must be optional, you should be able to run kubespray with no groups_vars at all. 

It's ok to re-define defaults here to highlight options a user probably wants to edit.